diff --git a/node_modules/react-native-hyperlink/src/Hyperlink.js b/node_modules/react-native-hyperlink/src/Hyperlink.js
index 40812a2..9216a8a 100644
--- a/node_modules/react-native-hyperlink/src/Hyperlink.js
+++ b/node_modules/react-native-hyperlink/src/Hyperlink.js
@@ -128,7 +128,7 @@ class Hyperlink extends Component {
 Hyperlink.propTypes = {
   linkDefault: PropTypes.bool,
   linkify: PropTypes.object,
-  linkStyle: textPropTypes.style,
+  linkStyle: Text.propTypes.style,
   linkText: PropTypes.oneOfType([
     PropTypes.string,
     PropTypes.func,
